atlas/AtlasMl/atlasml/ml/MLPipelines/PipelineWorkflows.py

```
@@ -0,0 +1,139 @@
+        similarity_scores (numpy.ndarray): Array of cosine similarity scores between the input embedding and each medoid.
+
+    Methods:
+        run(text: str, uuid: str) -> numpy.ndarray:
+            1. Generates a local embedding for the provided text and UUID.
+            2. Retrieves all cluster medoid embeddings from Weaviate.
+            3. Computes cosine similarity between the input embedding and each medoid.
+            4. Identifies the medoid with the highest similarity score.
+            5. Stores the input embedding in the TEXT collection with a reference to the selected medoid.
+            6. Returns the array of similarity scores.
+    """
+    def newTextPipeline(self, text: str, uuid: str):
```

Pylint (3.3.7)

[convention] Missing function or method docstring

(C0116)